lightning/src/util/config.rs

```diff
@@ -206,7 +206,21 @@ pub struct ChannelConfig {
 	/// This cannot be changed after a channel has been initialized.
 	///
 	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool
+	pub commit_upfront_shutdown_pubkey: bool,
+	/// Limit holder balance exposure to dusted HTLCs at any given time.
+	///
+	/// Applied on outgoing dust HTLCs and incoming dust HTLCs committed on the counterparty
+	/// commitment transaction.
+	///
+	/// Dust HTLCs are defined as any HTLC lower than counterparty's `dust_limit_satoshis` summed
+	/// up with their expected onchain claim cost at minimal mempools congestion.
+	///
+	/// Note, to support reading serialized configs before 0.0.100, the default value will be
```

Comment:
I think we can leave these for the CHANGELOG serialization compatibility section.